<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiConnectionWidget</class>
 <widget class="QWidget" name="WifiConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>279</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="SSIDLabel">
     <property name="text">
      <string>SSID:</string>
     </property>
     <property name="buddy">
      <cstring>SSIDCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="SsidComboBox" name="SSIDCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="modeLabel">
     <property name="text">
      <string>Mode:</string>
     </property>
     <property name="buddy">
      <cstring>modeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="modeComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Infrastructure</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ad-hoc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Access Point</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="BSSIDLabel">
     <property name="text">
      <string>BSSID:</string>
     </property>
     <property name="buddy">
      <cstring>BSSIDCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="BssidComboBox" name="BSSIDCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Restrict to device:</string>
     </property>
     <property name="buddy">
      <cstring>macAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="HwAddrComboBox" name="macAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Cloned MAC address:</string>
     </property>
     <property name="buddy">
      <cstring>clonedMacAddress</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KLineEdit" name="clonedMacAddress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMask">
        <string notr="true">HH:HH:HH:HH:HH:HH;_</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRandomMacAddr">
       <property name="text">
        <string>Random...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>MTU:</string>
     </property>
     <property name="buddy">
      <cstring>mtu</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="mtu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="specialValueText">
      <string>Automatic</string>
     </property>
     <property name="suffix">
      <string> bytes</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="hiddenNetwork">
     <property name="toolTip">
      <string>Mark this if you want to create a connection for a hidden network</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Mark this if you want to create a connection for a hidden network</string>
     </property>
     <property name="text">
      <string>Hidden network:</string>
     </property>
     <property name="buddy">
      <cstring>hiddenNetwork</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="bandLabel">
     <property name="text">
      <string>Band:</string>
     </property>
     <property name="buddy">
      <cstring>band</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KComboBox" name="band">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string extracomment="automatic band">Automatic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A (5 GHz)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B/G (2.4 GHz)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="channelLabel">
     <property name="text">
      <string>Channel:</string>
     </property>
     <property name="buddy">
      <cstring>channel</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="KComboBox" name="channel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>HwAddrComboBox</class>
   <extends>QComboBox</extends>
   <header>hwaddrcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>SsidComboBox</class>
   <extends>QComboBox</extends>
   <header>ssidcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>BssidComboBox</class>
   <extends>QComboBox</extends>
   <header>bssidcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SSIDCombo</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>BSSIDCombo</tabstop>
  <tabstop>band</tabstop>
  <tabstop>macAddress</tabstop>
  <tabstop>clonedMacAddress</tabstop>
  <tabstop>btnRandomMacAddr</tabstop>
  <tabstop>mtu</tabstop>
  <tabstop>hiddenNetwork</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
